### DISCUSSION TITLE
Cast giving segmentation fault

### DISCUSSION THREAD

**Discussion by Suraj7789** [Cyfrin/foundry-full-course-cu#1441] [Q&A]
Why Cast is giving segmentation fault even though all others forge, anvil, and chisel are running fine?

**Comment by EngrPips** [comment]
Hey @Suraj7789  giving more context about the issue you are having would really help us debug and maybe find solution.  for example 
=> what were you doing with cast before it returned segmentation result 
=> what output were you actually expecting to get back from cast 
and any more details would be of great help to help you debug

**Reply by Suraj7789** [reply]
I did nothing,
I was following the course and when i got to the cast part i ran 
cast --version and it was giving me segmentation fault
![Screenshot 2024-02-28 215515](https://github.com/Cyfrin/foundry-full-course-f23/assets/77788438/2d42a4cc-1891-4fb0-99c5-d54b1c4b0e6c)

**Reply by EngrPips** [reply]
Hey @Suraj7789  i am not sure why you was getting that error, i ran the same command on my machine and everything went fine. Do you still get the same error when you run `cast --version` ?

<img width="1080" alt="Screenshot 2024-02-29 at 1 18 14 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/63d7861b-cd8a-49a8-a641-6580b1f79741">
